File: src/my_cpp_pkg/src/template.cpp
#include "rclcpp/rclcpp.hpp"

class MyNode : public rclcpp::Node // MODIFY NAME
{
public:
    MyNode() : Node("cpp_test") // MODIFY NAME
    {
    }
private:
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<MyNode>(); // MODIFY NAME
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: my_py_pkg/my_py_pkg/template.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from robot_interfaces.msg import LEDStates

class LEDPanelNode(Node): # MODIFY NAME
    def __init__(self):
        super().__init__("led_panel") # MODIFY NAME


def main(args=None):
    rclpy.init(args=args)
    node = LEDPanelNode() # MODIFY NAME
    rclpy.spin(node)
    rclpy.shutdown()

if __name__ == "__main__":
    main()